agent_ignorelist.yaml

```
@@ -64,6 +64,12 @@ ignorelist:
     matcher: redhat
     skip_if: "{{ (version == '5.14.0' and (rpmrelver|int) >= 410) }}"
 
+  - description: "[SMAGENT-8627] kernel 5.14 - RHEL backport: mnt_idmap for fs"
+    probe_versions: [ 13.4.0, 13.4.1, 13.5.0, 13.6.0, 13.6.1, 13.7.0, 13.7.1, 13.7.2 ]
+    probe_kinds: [ kmod ]
+    matcher: redhat
+    skip_if: "{{ (version == '5.14.0' and (rpmrelver|int) >= 547) }}"
```

```suggestion
    skip_if: "{{ (version == '5.14.0' and (rpmrelver|int) >= 542) }}"
```
Since there's also a 5.14.0-542 for which only `kernel-devel` exists (reasons unclear, at least to me) so the build fails anyway.